src/Workspaces/Core/Portable/PatternMatching/AllLowerCamelCaseMatcher.cs

```diff
@@ -134,6 +141,21 @@ private PatternMatchKind GetKind(CamelCaseResult result)
                 return bestResult;
             }
 
+            private static char ToLower(char v, TextInfo textInfo)
+            {
+                return IsAscii(v)
```

Comment:
❓ `TextInfo.ToLower` already has this fast path internally. Given that we now cache the `TextInfo` instance, what is the incremental benefit of needing to maintain a copy of this handling?